draft-ietf-tls-dtls13.md

```diff
@@ -1594,8 +1594,10 @@ record_numbers:
   order.
 
 Implementations MUST NOT acknowledge records containing
-non-duplicative handshake messages or fragments which have not been
+handshake messages or fragments which have not been
 processed or buffered. Otherwise, deadlock can ensue.
+As an example, implementations MUST NOT send ACKs for
+handshake messages which they discard as out-of-order.
```

Comment:
This is actually different. Consider the case where I receive:

```
1: MSG=1 [1:100]
2: MSG=2 [1:100]
3: MSG=1 [1:50]
```
I may discard three because it's duplicative and/or out of order, but I should ACK it.

Reply:
Ah, that's what's meant here - @thomas-fossati and I were pondering for a bit, but didn't get the intended meaning of non-duplicative. I agree that in your example, msg 3 should be ACKed.

I think the issue here is the ambiguity of what 'processed' means - I'd say that message 3 will still be 'processed', even though it will be noted that its content doesn't reveal anything new.

Maybe we can find a wording which is less prone to different interpretations.